packages/tools/generator-component/generators/app/templates/src/components/Skeleton.vue

```diff
@@ -1,36 +1,36 @@
 <template>
     <div
-        :data-theme="theme"
         :class="$style['c-<%= name.class %>']"
-        data-test-id='<%= name.class %>-component'>
```

Comment:
This change breaks the webdriver io tests

Reply:
Will update the tests

Reply:
Done